GovUk.Frontend.Umbraco.ExampleApp/GovUk.Frontend.Umbraco.ExampleApp.csproj

```diff
@@ -66,6 +66,7 @@
     </ItemGroup>
     <ItemGroup>
       <Folder Include="App_Plugins\GOVUK\blocks\controllers\" />
+      <Folder Include="uSync\v9\Blueprints\" />
```

Comment:
You removed the blueprint files but this reference to including the folder in the project needs to go too.